lib/game/widget_builder.dart

```diff
@@ -68,19 +85,18 @@ Widget _applyAdvancedGesturesDetectors(Game game, Widget child) {
     );
   }
 
-  if (game is MultiTouchDragDetector) {
+  if (_dragHandlers.isNotEmpty) {
     gestures[ImmediateMultiDragGestureRecognizer] =
         GestureRecognizerFactoryWithHandlers<
             ImmediateMultiDragGestureRecognizer>(
       () => ImmediateMultiDragGestureRecognizer(),
-      (ImmediateMultiDragGestureRecognizer instance) {
+      (MultiDragGestureRecognizer instance) {
         instance
           ..onStart = (Offset o) {
             final drag = DragEvent();
+            // TODO: Remove any padding/margin on the widget here
```

Comment:
Should we though? I think we can leave this to programmer?

Reply:
Then we would have inconsistencies in the API since they are removed for `Tapable`.

Reply:
I don't think there is a good way to do it so I added a comment in the code and a note in the docs about the behavior.